architecture/rbac.md

```diff
@@ -0,0 +1,49 @@
+# RBAC
+
+![RBAC](images/rbac.svg)
+
+## How it works
+
+RBAC in ManageIQ is split into 2 parts; what you can "see" and what you can "do".  All user interaction first pass through the first layer of things you can "see" based on a number of criteria.  On those objects, then, only a certain subset of actions are allowed based on what you can "do".
```

Comment:
>All user interaction first pass through the first layer of things you can "see" based on a number of criteria

I'm not sure what you're trying to say here with wit the use "first" and "layers".  99% of user interaction involves users being shown objects they are allowed to see.  We then enable/disable various actions in menus based on what you can do.

Reply:
Yeah good point.  Later, the "layers" are enumerated, but here it's premature.  I'll reword.